### PR TITLE
sort and actions

### PR DESCRIPTION
**Pull request by ramitos**

### CODE REVIEW

File: packages/my-joy-beta/src/containers/instances/firewall.js [hunk 1]
```diff
@@ -32,13 +32,14 @@ const Firewall = ({
...
-      title="Ooops!"
-      message="An error occurred while loading your instance firewall rules"
-      error
-    />
-  ) : null;
+  const _error =
+    error && !values.length && !_loading ? (
+      <Message
+        title="Ooops!"
+        message="An error occurred while loading your instance firewall rules"
```
Comment (SaraVieira) [inline packages/my-joy-beta/src/containers/instances/firewall.js]: Use as children pls

File: packages/my-joy-beta/src/containers/instances/metadata.js [hunk 2]
```diff
@@ -38,19 +38,18 @@ const MetadataForms = (metadata = []) =>
...
-      title="Ooops!"
-      message="An error occurred while loading your instance metadata"
-      error
-    />
-  ) : null;
+  const _error =
+    error && !values.length && !_loading ? (
+      <Message
+        title="Ooops!"
+        message="An error occurred while loading your instance metadata"
```
Comment (SaraVieira) [inline packages/my-joy-beta/src/containers/instances/metadata.js]: Same here

File: packages/my-joy-beta/src/containers/instances/networks.js [hunk 3]
```diff
@@ -28,13 +28,14 @@ const Networks = ({ networks = [], loading, error }) => {
...
-      title="Ooops!"
-      message="An error occurred while loading your instance networks"
-      error
-    />
-  ) : null;
+  const _error =
+    error && !values.length && !_loading ? (
+      <Message
+        title="Ooops!"
+        message="An error occurred while loading your instance networks"
```
Comment (SaraVieira) [inline packages/my-joy-beta/src/containers/instances/networks.js]: Same

File: packages/my-joy-beta/src/containers/instances/tags.js [hunk 4]
```diff
@@ -38,19 +38,18 @@ const TagForms = (tags = []) =>
...
-      title="Ooops!"
-      message="An error occurred while loading your instance tags"
-      error
-    />
-  ) : null;
+  const _error =
+    error && !values.length && !_loading ? (
+      <Message
+        title="Ooops!"
+        message="An error occurred while loading your instance tags"
```
Comment (SaraVieira) [inline packages/my-joy-beta/src/containers/instances/tags.js]: Same

File: packages/my-joy-beta/src/containers/instances/firewall.js (hunk 1 above)
Comment (ramitos) [reply packages/my-joy-beta/src/containers/instances/firewall.js]: Ah, thanks. Forgot that the API changed ;)

**Pull request merged**